lte/gateway/c/core/oai/tasks/amf/amf_authentication.cpp

```diff
@@ -121,31 +119,66 @@ static int start_authentication_information_procedure(
   auth_info_proc->ue_id  = ue_id;
   auth_info_proc->resync = auth_info_proc->request_sent;
 
-  bool is_initial_req          = !(auth_info_proc->request_sent);
-  auth_info_proc->request_sent = true;
+  auth_proc->auts = bstrcpy(auts);
+
+  rc = amf_authentication_request_sent(ue_id);
+  if (rc != RETURNok) {
+    OAILOG_FUNC_RETURN(LOG_NAS_AMF, rc);
+  }
 
+  OAILOG_FUNC_RETURN(LOG_NAS_AMF, RETURNok);
+}
+
+int amf_authentication_request_sent(amf_ue_ngap_id_t ue_id) {
+  OAILOG_INFO(
+      LOG_AMF_APP, "inside amf_authentication_request_sent");  // need to remove
+  ue_m5gmm_context_s* ue_mm_context = NULL;
+  amf_context_t* amf_context        = NULL;
+  char imsi_str[IMSI_BCD_DIGITS_MAX + 1];
+  int rc           = RETURNerror;
+  ue_mm_context    = amf_ue_context_exists_amf_ue_ngap_id(ue_id);
+  uint8_t snni[40] = {0};
+  if (!ue_mm_context) {
+    OAILOG_WARNING(
+        LOG_NAS_AMF,
+        "AMF-PROC - Failed authentication request the UE due to NULL"
+        "ue_mm_context\n");
```

Comment:
Instead of displaying the exact variable name "ue_mm_context", use words to describe it.

Reply:
comment addressed.